gradle.properties

```diff
@@ -4,24 +4,24 @@ pluginGroup = intellij
 pluginName = intellij-buf
 pluginRepositoryUrl = https://github.com/bufbuild/intellij-buf
 # SemVer format -> https://semver.org
-pluginVersion = 0.4.3
+pluginVersion = 0.5.0
 
 # Supported build number ranges and IntelliJ Platform versions -> https://plugins.jetbrains.com/docs/intellij/build-number-ranges.html
-pluginSinceBuild = 223
-pluginUntilBuild = 243.*
+pluginSinceBuild = 243
+pluginUntilBuild = 253.*
```

Comment:
Should this be `251`? I thought your message in Slack suggested we can't do this since this isn't an actual version yet.

Reply:
I think we have options - either we can remove this property and claim compatibility with all future versions, or we can apply this pattern and claim compatibility with 2025.* versions until we bump again.

I think this is just in the metadata which is published with the plugin. In order for the plugin to be published it has to pass verification tests against released versions.